twake/frontend/src/app/services/languages/locale/fr.js

```diff
@@ -85,14 +85,16 @@ export default {
     'Société, club ou association',
   'scenes.app.workspaces.create_company.group_data.group_type.other_group': 'Autre',
   'scenes.app.workspaces.create_company.group_data.group_size': 'Taille du groupe',
-  'scenes.app.workspaces.create_company.group_data.group_size.less_3': 'Moins de 4 personnes',
-  'scenes.app.workspaces.create_company.group_data.group_size.4_10': '4 - 10 personnes',
-  'scenes.app.workspaces.create_company.group_data.group_size.10_50': '11 - 49 personnes',
-  'scenes.app.workspaces.create_company.group_data.group_size.50_100': '50 - 99 personnes',
-  'scenes.app.workspaces.create_company.group_data.group_size.100_500': '100 - 499 personnes',
-  'scenes.app.workspaces.create_company.group_data.group_size.500_1000': '500 - 999 personnes',
-  'scenes.app.workspaces.create_company.group_data.group_size.1000_5000': '1000 - 4999 personnes',
-  'scenes.app.workspaces.create_company.group_data.group_size.5000_more': 'Plus de 5000 personnes',
+  'scenes.app.workspaces.create_company.group_data.group_size.less_3': 'Moins de 4 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.4_10': '4 - 10 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.10_50': '11 - 49 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.50_100': '50 - 99 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.100_500': '100 - 499 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.500_1000': '500 - 999 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.1000_5000':
+    '1000 - 4999 utilisateurs',
+  'scenes.app.workspaces.create_company.group_data.group_size.5000_more':
+    'Plus de 5000 utilisateurs',
```

Comment:
@BenoitTallandier for this ones maybe not great?

Reply:
This one is ok, it's by users so I would keep "utilisateur" :)